### DISCUSSION TITLE
Wen Änderungen an CP gemacht worden

### DISCUSSION THREAD

**Discussion by Legion68** [Courseplay/Courseplay_FS22#2997] [Q&A]
Sind dann die Änderungen beim DL dann hier mit drin oder muss man warten biss Neue Version hier zum DL kommt bei GitHub  ?

**Comment by Tensuko** [comment]
Warte bitte bis die neue Version da ist. (Wird dann die 7.4.0.1)

**Reply by Legion68** [reply]
OK alles klar Danke Dir für die schnelle Antwort
